src/main/java/ru/sbtqa/tag/qautils/i18n/I18N.java

```diff
@@ -1,28 +1,28 @@
 package ru.sbtqa.tag.qautils.i18n;
 
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
-
 import java.io.IOException;
 import java.io.InputStream;
 import java.io.InputStreamReader;
-
+import java.util.HashMap;
 import java.util.Locale;
 import java.util.Map;
 import java.util.Properties;
 import java.util.Set;
-import java.util.HashMap;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 /**
  * i18n resource bundle from UTF-8 properties
  */
 public class I18N {
 
     private static final Logger LOG = LoggerFactory.getLogger(I18N.class);
+    
     private static final Map<String, I18N> BUNDLE_STORAGE = new HashMap<>();
     private static final String BUNDLE_ENCODING = "UTF-8";
     public static final String DEFAULT_BUNDLE_PATH = "i18n";
     public static final String SECRET_DELIMITER = "°\u0000\u0000\u0000 ";
+    private static final String DELIMITER = "/";
```

Comment:
Is is safe to use hardcoded delimiter on different OS?

Reply:
nobody complained

Reply:
how about FileSystems.getDefault().getSeparator()?

Reply:
No, because this is about resource files, we don't want to use FileSystems.getDefault().getSeparator() at all. The defined path separator is /. It is not platform-dependent.